scss/_patterns_links.scss

```diff
@@ -31,6 +31,24 @@
     }
   }
 
+  .p-link--skip {
+    color: $color-link;
+    display: block;
+    left: -999px;
+    position: absolute;
+    top: -999px;
+
+    &:focus {
```

Comment:
Why not use the focus placeholder here? If the focus style changes this would need to be updated too.